doc/README.md

```diff
@@ -1,3 +1,19 @@
+## Deprecation Notice
+
+The new open source [Citus extension](https://github.com/citusdata/citus)
+for PostgreSQL natively integrates `pg_shard` and provides a superset
+of its functionality.  We’ll not be making any more `pg_shard`
+releases and the project is entering end-of-life status.
+
+We have migration steps to help existing customers perform a drop-in
+replacement. [Contact
+us](mailto:engage@citusdata.com?subject=pg_shard+Migration) for
+more information, or feel free to drop in to our [IRC
```

Comment:
Replace _drop in_ with _stop by_. The proximity of the phrase to _drop-in replacement_ felt weird to me.